### PR TITLE
32 behat chrome

### PR DESCRIPTION
**Pull request by mcurran-inviqa**

### CODE REVIEW

File: src/drupal8/application/skeleton/composer.json [hunk 1]
```diff
@@ -13,18 +13,14 @@
...
-      "url": "https://github.com/my127/behat-chrome-extension"
     }
   },
   "require-dev": {
     "webflo/drupal-core-require-dev": "^8",
     "drupal/devel": "^2.0",
     "drupal/drupal-extension": "^3.4",
     "mikey179/vfsstream": "^1.0",
-    "my127/behat-chrome-extension": "^1.2"
+    "dmore/behat-chrome-extension": "^1.3"
```
Comment (kierenevans) [inline src/drupal8/application/skeleton/composer.json]: Please reorder to be alphabetical (same for all)

**Review by kierenevans — CHANGES_REQUESTED** [review]

File: src/drupal8/application/skeleton/composer.json (hunk 1 above)
Comment (tkotosz) [reply src/drupal8/application/skeleton/composer.json]: maybe use
```
"config": {
        "sort-packages": true
    },
```
https://getcomposer.org/doc/06-config.md#sort-packages
?

Comment (tkotosz) [reply src/drupal8/application/skeleton/composer.json]: ah it is already used :D I guess the composer.json was edited manually :D

**Review by kierenevans — APPROVED** [review]
Thanks!

**Pull request merged**